src/common/types/generic.ts

```
@@ -37,10 +37,15 @@ export const illinoisUin = z
     id: "IllinoisUin",
   });
 
+export const NonAcmOrgUniqueId = z.enum(AllOrganizationIdList.filter(x => x !== "A01")).meta({
```

ESLint

[error] Replace `.enum(AllOrganizationIdList.filter(x·=>·x·!==·"A01"))` with `⏎··.enum(AllOrganizationIdList.filter((x)·=>·x·!==·"A01"))⏎··`

(prettier/prettier)